packages/flutter_tools/lib/src/commands/version.dart

```diff
@@ -45,7 +45,7 @@ class VersionCommand extends FlutterCommand {
     RunResult runResult;
     try {
       runResult = await processUtils.run(
-        <String>['git', 'tag', '-l', 'v*', '--sort=-creatordate'],
+        <String>['git', 'tag', '-l', '*.*.*', '--sort=-creatordate'],
         throwOnError: true,
         workingDirectory: Cache.flutterRoot,
       );
```

Comment:
Looking at the toolExit below. If git is not installed, I think the exception would be an `ArgumentError` and not a `ProcessException`.

Reply:
should the action point be to remove the reference to git not being installed? I was gonna check if git `canRun`, but then i realize if they don't have git installed they'll never get to this point, as it would fail earlier.

as for how to handle the case of the user not having git installed, I'd rather have a flutter tool-wide solution for https://github.com/flutter/flutter/issues/52262

Reply:
Yeah, removing the reference to git not being installed sgtm.

Reply:
Done